woocommerce/woocommerce
# Add/35129 product mvp ces

### All Submissions:






### Changes proposed in this Pull Request:

Add's a CES bar to the footer of the Product MVP page when a user saves or publishes a product for the first time.

Closes #35129 

https://user-images.githubusercontent.com/2240960/203027445-7963d073-37a6-45ee-9e11-7a54678cd03c.mp4






### How to test the changes in this Pull Request:



1. Load this branch, build it, and make sure you have the `new-product-management-experience` feature flag enabled (you can do so using the latest version of the Beta tester within the mono repo).
2. Make sure you have tracks enabled and also outputted in the console: localStorage.setItem( 'debug', 'wc-admin:*' );
3. Go to Products > Add New (MVP)
4. Add a name and click Save draft
5. A bar in the footer should show up saying: `You're using the new product editor (currently in development). How is your experience so far?` and give the option to share feedback
6. Refresh the page, the bar should show up again after the product has been loaded.
7. When clicking share feedback the CES modal should show up. You can fill it out and click Send
8. This should trigger a track event with `new_product` action and the score.
9. The bar should stay put after submitting feedback
10. Now click Publish and enter feedback again, this should trigger the `new_product` action once finished.
11. The feedback footer should still show up after, even after refresh.
12. Now click the x or `Turn it off` on the notice, the notice should not show up there after, also on refresh.

If you wanted to re-test you can delete the `woocommerce_ces_product_mvp_ces_action` option.



### Other information:




### FOR PR REVIEWER ONLY:
